tests/nexus/workflow_test.py

```diff
@@ -519,6 +558,21 @@ def test_load_monitor_workflow() -> None:
     assert 'source_position' in da.coords
     assert da.bins is not None
     assert da.dims == ('event_time_zero',)
+    assert da.bins.constituents['data'].variances is not None
+
+
+def test_load_histogram_monitor_workflow() -> None:
+    wf = LoadMonitorWorkflow(run_types=[SampleRun], monitor_types=[FrameMonitor1])
+    wf[Filename[SampleRun]] = data.dream_coda_test_file()
+    wf[NeXusName[FrameMonitor1]] = 'monitor_bunker'
+    da = wf.compute(MonitorData[SampleRun, FrameMonitor1])
+    assert 'position' in da.coords
+    assert 'source_position' in da.coords
+    assert da.bins is None
+    assert da.dims == ('time', 'frame_time')
```

Comment:
```suggestion
    assert set(da.dims) == {'time', 'frame_time'}
```
Could the order of dims change?

Reply:
Possibly. And we don't need to rely on a specific order :+1: